horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/datastore/PostgresDatastoreConfig.java

```diff
@@ -3,9 +3,20 @@
 import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
+import io.hyperfoil.tools.horreum.api.data.datastore.auth.NoAuth;
+
 @Schema(type = SchemaType.OBJECT, required = true, description = "Built in backend datastore")
 public class PostgresDatastoreConfig extends BaseDatastoreConfig {
 
+    public static final String[] auths = { NoAuth._TYPE };
```

Comment:
```suggestion
```

Reply:
This is accessed via reflection: https://github.com/Hyperfoil/Horreum/pull/2223/files#diff-8a4bd7c5a2730d0b066069e4fe1d81fd44eb3c7f84b69867ad74b83f74aca8deR27-R30